sources/packages/web/src/types/AppRoutes.ts

```diff
@@ -28,6 +28,7 @@ export enum AppRoutes {
   InstitutionManageUsers = "manage-users",
   DisabledUser = "login/disabled-user",
   StudentApplication = "application-form/:selectedForm/programYear/:programYearId/application/:id",
+  StudentApplicationView = "application-form/:selectedForm/programYear/:programYearId/application/:id/:readOnly",
```

Comment:
Do we need a new route for this? Maybe we just need to change the ":readOnly" to be optional or, if there will be a different route, do we need ":readOnly" as a parameter?

Reply:
I created it optional, but do not want mess with the edit route as its used for more than one place.

Reply:
For the first point "I created it optional, but do not want mess with the edit route as its used for more than one place", it will not mess with current routes, it is backward compatible 😉 

For the second point "I prefer to have it 2 and pass the parameter, as we can use it in the form to show/hide the buttons", the property name is readOnly and it is receiving a string "readOnly" that is not being used in any check. The check is based in the fact that there is something or not in the parameter. So, if I provide "anything" in the readOnly parameter it will assume that the form will be readonly, right?

Reply:
I agree with the first point, the second point I intended to create that way. Is that wrong?

Reply:
I would not say wrong, but to me, it is weird that I would access the URL like below and the form will be readonly 😉 
```
application-form/THEFROM/programYear/1/application/126/I_WANT_THE_FORM_TO_BE_EDITABLE
```

Reply:
Please do not consider this issue as a blocker.